src/app/shared/services/schema-keys-store.service.ts

```diff
@@ -86,10 +88,14 @@ export class SchemaKeysStoreService {
       if (subSchema['type'] === 'object') {
         if (subSchema['properties'][splitPath[index]]) {
           subSchema = subSchema['properties'][splitPath[index]];
+        } else {
+          return null;
```

Comment:
why is that necessary?

Reply:
If a key is invalid im returning a null value allowing me to know that the path was faulty